src/logic/queryParams.js

```diff
@@ -35,7 +36,22 @@ import {
  * @return {string}
  */
 const getSearchQueryFromQueryParameterOrLocalStorage = route => {
-  const { [CommonQueryParameters.SearchFilters]: sq } = route?.query;
+  const {
+    [CommonQueryParameters.SearchFilters]: sq,
+    [CommonQueryParameters.PlainSearchFilters]: f,
```

Comment:
I would recommend extracting filters/query search from `f` as an exception on the `SearchPage`. Having it here sends a wrong message that we support both query parameters even on the pages that never supported `f` parameter before, when in fact this is just a backward compatibility with old versions. Without even mentioning that having it here makes this function more complex.

Reply:
updated! I prefer to have it here so that even serachImage can take advantage of that